chat/message/src/Message/Message.tsx

```diff
@@ -148,3 +160,39 @@ export const Message = forwardRef(
 Message.displayName = 'Message';
 
 export default Message;
+
+function VerifiedAnswerBanner({
+  verifier,
+  verifiedAt,
+  learnMoreUrl,
+}: NonNullable<MessageProps['verified']>) {
+  const text = useMemo(() => {
+    const textParts = [`Answer Verified`];
+
+    if (verifier) {
+      textParts.push(`By ${verifier}`);
+    }
+
+    if (verifiedAt) {
+      const formattedDate = verifiedAt.toLocaleDateString('en-US', {
+        year: 'numeric',
+        month: 'long',
+        day: 'numeric',
+      });
+      textParts.push(`On ${formattedDate}`);
```

Comment:
Is this the desired order of data? (i.e. "Answer verified by ABC on XYZ"

Reply:
Also, I believe this should be sentence case
![Screenshot 2024-04-02 at 11 40 53 AM](https://github.com/mongodb/leafygreen-ui/assets/2414030/6e58dc48-2cb3-4c43-8697-cb226c05a868)

Reply:
Designs use title case but I agree sentence case is reasonable. I'll run it by Alven and switch.

Reply:
Alven says the title case is intentional and wants to stick with it here.

Reply:
Also yeah this is the order we use in the design `Answer Verified By <verifier> On <date>`